tests/integration-tests/Android/Android.CLI.Commands.Tests.proj

```diff
@@ -0,0 +1,80 @@
+<Project DefaultTargets="Test">
+  <Import Sdk="Microsoft.DotNet.Helix.Sdk" Project="Sdk.props"/>
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <!-- Workaround changes from newer MSBuild requiring additional properties, see https://github.com/dotnet/arcade/pull/5996 -->
+    <TargetFrameworkVersion>3.1</TargetFrameworkVersion>
+    <TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
+    <HelixType>test/product/</HelixType>
+    <IncludeXHarnessCli>true</IncludeXHarnessCli>
+    <TestRunNamePrefix>$(AGENT_JOBNAME)</TestRunNamePrefix>
+    <EnableAzurePipelinesReporter>true</EnableAzurePipelinesReporter>
+    <HelixBaseUri>https://helix.dot.net</HelixBaseUri>
+    <!-- Pick up the nupkg from this repo for testing purposes -->
+    <MicrosoftDotNetXHarnessCLIVersion>1.0.0-ci</MicrosoftDotNetXHarnessCLIVersion>
+    <XHarnessNupkgPath>$(MSBuildThisFileDirectory)../../../artifacts/packages/$(Configuration)/Shipping/Microsoft.DotNet.XHarness.CLI.$(MicrosoftDotNetXHarnessCLIVersion).nupkg</XHarnessNupkgPath>
+
+    <!-- Required: Version of XHarness CLI to use -->
+    <IncludeXHarnessCli>true</IncludeXHarnessCli>
+
+    <!-- Optional: Properties that are also valid for the Arcade Helix SDK (some might be needed for CI runs only) -->
```

Comment:
Looks like properties from Optional and lower are duplicate, so I would remove these